.eslintrc

```diff
@@ -0,0 +1,91 @@
+{
+	"env": {
+		"browser": true,
+		"node": true,
+	},
+	"extends": "eslint:recommended",
+	"globals": {
+		"Prism": "readonly",
+	},
+	"parserOptions": {
+		"ecmaVersion": 5,
+	},
+	"root": true,
+	"rules": {
+//		"indent": [
+//			"error",
+//			"tab",
+//			{
+//				"SwitchCase": 1,
+//				"outerIIFEBody": 0,
+//			}
+//		],
+		"linebreak-style": [
+			"error",
+			"unix",
+		],
+		"no-console": "off",
+		"no-empty": "off",
+		"no-inner-declarations": "off",
+		"no-mixed-spaces-and-tabs": "error",
+		"no-redeclare": "off",
+		"no-sparse-arrays": "off",
+		"no-unused-vars": [
+			"warn",
+			{
+				"args": "none",
+				"argsIgnorePattern": "^_",
+				"varsIgnorePattern": "^_",
```

Comment:
There are no unused variables which match this pattern.

Reply:
There&nbsp;is&nbsp;now:

https://github.com/PrismJS/prism/blob/3fb78077e23ddc2cdd102506d1a43735764e329e/code.js#L21

Reply:
Hmmm, unused variables are usually a sign that something went wrong during refactoring or writing, so why would we want to ignore these?

(`_`)`after` is a good example: Why is it there? Does it do anything?
`nextSibling` should be side-effect free, so why _not_ remove it. I don't see the benefit in ignoring a potential mistake because of its name.
(I mean, even after the ESLint warning disappears, I have like 3 other linters which all scream at me to remove it...)

Also, even after you remove the ignore pattern, `after` stays the only unused variable.

Reply:
I think we can just remove the unused variable.

Reply:
Can't we remove the ignore pattern now or is there a reason to keep it?

Reply:
Agreed, remove the ignore pattern.